java/vector/src/main/java/org/apache/arrow/vector/GenerateSampleData.java

```diff
@@ -0,0 +1,337 @@
+/*******************************************************************************
```

Comment:
Don't know if it's github issue, is the license format correct?

Reply:
Corrected. Not a github issue.

Reply:
@siddharthteotia The license format still doesn't look correct.